ifrs17/Import/Importers.ipynb

```diff
@@ -586,88 +632,83 @@
                 "\n    await storage.InitializeAsync();",
                 "\n    if(Activity.HasErrors()) return Activity.Finish();",
                 "\n       ",
-                "\n    var errors = new List<string>();",
+                "\n    var errors = new List<string>();   ",
```

Comment:
Is this used?

Reply:
I agree, it is not used anymore.  Deleted